### PR TITLE
[PM-368] Fix for sort direction UI

### PR DESCRIPTION
**Pull request by MattHallEA**
Pass the sort order and column through to the view

### CODE REVIEW

File: app/views/pafs_core/projects/index.html.erb [hunk 1]
```diff
@@ -38,7 +38,7 @@
             <th scope="col" class="sortable-heading">Project name</th>
             <th scope="col">Created by</th>
             <th scope="col">
-              <% sort_properties_for_col = get_next_sort_order_and_curr_arrow(@project_sort_column, "updated_at", @project_sort_order, "updated_at", "desc") %>
```
Comment (baphled) [inline app/views/pafs_core/projects/index.html.erb]: Could we move this into another PR?

We could do this as one piece of house keeping and keep the changes contained to one PR :)

Comment (MattHallEA) [reply app/views/pafs_core/projects/index.html.erb]: Not really - this is part of the fix :)

Comment (baphled) [reply app/views/pafs_core/projects/index.html.erb]: Cool.

**Comment by baphled** [comment]
:shipit:
